js/cards.js

```diff
@@ -40,11 +40,13 @@ $(document).ready(() => {
         renderCard(data);
         countCards();
         renderIfUserHasRaised(data.id);
+        if (uid && data.id == uid) toggleCardButton(data.card);
         break;
       case "lower":
         $(`#${data.id}${data.card}`).remove();
         countCards();
         renderIfUserHasRaised(data.id);
+        if (uid && data.id == uid) toggleCardButton(data.card);
```

Comment:
Just toggling the cards here assumes a specific previous state which may not be correct, although the state to be set is known here. Instead of toggling you should explicitly set the target state of the cards:

```suggestion
        if (uid && data.id == uid) setCardButton(data.card, true);
```

Reply:
this still applies